ipecharts/__init__.py

```diff
@@ -14,5 +14,28 @@
     __version__ = "dev"
 
 
+def _prefix():
+    import sys
+    from pathlib import Path
+
+    prefix = sys.prefix
+    here = Path(__file__).parent
+    # for when in dev mode
+    if (here.parent / "prefix").exists():
+        prefix = str(here.parent)
+    return prefix
+
+
 def _jupyter_labextension_paths():
     return [{"src": "labextension", "dest": "ipecharts"}]
+
+
+def _jupyter_nbextension_paths():
+    return [
+        {
+            "section": "notebook",
+            "src": f"{_prefix()}/prefix/share/jupyter/nbextensions/jupyter-vuetify/",
```

Comment:
```suggestion
            "src": f"{_prefix()}/prefix/share/jupyter/nbextensions/ipecharts/",
```

There are some other `vuetify` references in the PR